catalog/dags/database/batched_update/batched_update_dag.py

```diff
@@ -51,23 +49,18 @@
 }
 ```
 
-It is possible to resume an update from an arbitrary starting point on an existing
-temp table, for example if a DAG succeeds in creating the temp table but fails midway
-through the update. To do so, set the `resume_update` param to True and select your
-desired `batch_start`. For instance, if the example DAG given above failed after
-processing the first 50_000 records, you might run:
-
-```
-{
-    "query_id": "my_flickr_query",
-    "table_name": "image",
-    "select_query": "WHERE provider='flickr'",
-    "update_query": "SET thumbnail=null",
-    "batch_size": 10,
-    "batch_start": 50000,
-    "resume_update": true,
-    "dry_run": false
-}
+The `update_batches` task automatically keeps track of its progress in an Airflow
+variable suffixed with the `query_id`. If the task fails, when it resumes (either
+through a retry or by being manually cleared), it will pick up from where it left
+off. Manually managing this Airflow variable should not be necessary.
+
+It is also possible to start an entirely new DagRun using an existing temp table,
+by setting the `resume_update` param to True. With this option enabled, the
+DAG will skip creating the temp table and instead attempt to run an update with an
+existing temp table matching the `query_id`. This option should only be used when
+the DagRun configuration needs to be changed after the table was already created:
+for example, if there was a problem with the `update_query` which caused DAG
+failures during the `update_batches` step.
 ```
```

Comment:
Oh! @stacimc it looks like this erroneous triple backtick is throwing the linting off because it thinks it's in a code block 😅

Reply:
Ah! Good find, that explains a lot 😄